### ISSUE TITLE
Code colors have insufficient contrast

### ISSUE THREAD

**Issue by JasonWeill** [jupyterlab/jupyterlab#14255]
## Description

Some colors used to render code in both Light and Dark Theme don't have enough contrast vs background (should be 7:1)

![contrast](https://user-images.githubusercontent.com/93281816/226788366-585d2a2d-c205-4d60-8820-75093d9f8374.png)

This was found during a bug bash by @andrii-i .
